doc/dev/crimson/crimson.rst

```diff
@@ -182,6 +182,9 @@ using ``vstart.sh``,
 ``--memstore``
     use the alienized MemStore as the object store backend.
 
+``--seastore``
+    use the SeaStore as the object store backend.
```

Comment:
s/use the SeaStore/use SeaStore/

Remove "the". That is, if I understand correctly what Seastore is. The definite article should be removed from this sentence.

Reply:
Yes, I have modified, thanks.